dotnet-authserver/src/TeacherIdentity.AuthServer/TeacherIdentity.AuthServer.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="EFCore.NamingConventions" Version="7.0.2" />
     <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="11.5.1" />
     <PackageReference Include="Flurl" Version="3.0.7" />
-    <PackageReference Include="GovUk.Frontend.AspNetCore" Version="1.0.0-beta4" />
+    <PackageReference Include="GovUk.Frontend.AspNetCore" Version="1.0.0-rc1" />
```

Comment:
This version exposes the built-in `DateOnly` model binder so we can layer it with our own.